index.html

```diff
@@ -0,0 +1,111 @@
+            <div class="form-check my-2">
+                <label class="form-check-label col-md-12 my-2" for="option-1"><input checked class="form-check-input" id="option-1" value="Definitely" type="radio"
+                        name="recomandation" />Definitely</label>
+                <label class="form-check-label col-md-12 my-2" for="option-2"><input class="form-check-input" id="option-2" value="Maybe" type="radio"
+                        name="recomandation" />
+                    Maybe</label>
+                <label class="form-check-label col-md-12 my-2" for="option-3"><input class="form-check-input" id="option-3" value="Not sure" type="radio"
+                        name="recomandation" />Not sure</label>
+            </div>
+            <p class="mb-0">What would you like to see improved? (Check all that apply)</p>
+            <div class="form-check my-2">
+                <label class="form-check-label col-md-12 my-2" for="check-1"><input class="form-check-input" id="check-1" value="Front-end Projects"
+                        type="checkbox" />Front-end Projects</label>
+                <label class="form-check-label col-md-12 my-2" for="check-2"><input class="form-check-input" id="check-2" value="Back-end Projects"
+                        type="checkbox" />Back-end Projects</label>
+                <label class="form-check-label col-md-12 my-2" for="check-3"><input class="form-check-input" id="check-3" value="Data Visualization"
+                        type="checkbox" />Data Visualization</label>
+                <label class="form-check-label col-md-12 my-2" for="check-4"><input class="form-check-input" id="check-4" value="Open Source Community"
+                        type="checkbox" />Open Source Community</label>
+                <label class="form-check-label col-md-12 my-2" for="check-3"><input class="form-check-input" id="check-3" value="Gitter help rooms"
+                            type="checkbox" />Gitter help rooms</label>
+                <label class="form-check-label col-md-12 my-2" for="check-4"><input class="form-check-input" id="check-4" value="Videos"
+                            type="checkbox" />Videos</label>
+                <label class="form-check-label col-md-12 my-2" for="check-4"><input class="form-check-input" id="check-4" value="City Meetups"
+                                type="checkbox" />City Meetups</label>
+                <label class="form-check-label col-md-12 my-2" for="check-4"><input class="form-check-input" id="check-4" value="Wiki"
+                                    type="checkbox" />Wiki</label>
+                <label class="form-check-label col-md-12 my-2" for="check-4"><input class="form-check-input" id="check-4" value="Forum"
+                                        type="checkbox" />Forum</label>
+                <label class="form-check-label col-md-12 my-2" for="check-4"><input class="form-check-input" id="check-4" value="Additional Courses"
```

Comment:
- [ ] You have duplicates of `input` fields of the same id `check-3` and `check-4`. Please provide distinct `id` for each input fields